resources/views/platform/components/game/similar-game-table-row.blade.php

```diff
@@ -28,15 +28,15 @@
                     points
                 </p>
 
-                <x-points-weighted-container>
-                    <p class="text-2xs whitespace-nowrap {{ $totalRetroPoints === 0 ? 'text-text-muted' : '' }}">
+                <p class="text-2xs whitespace-nowrap {{ $totalRetroPoints === 0 ? 'text-text-muted' : '' }}">
+                    <x-points-weighted-container>
```

Comment:
Moved this around because points-weighted-container's root element was changed from a div to a span. As a text element, it makes more semantic sense for it to be housed in a span (and makes adding it to complex layouts much less frustrating).